workspaces/copilot/plugins/copilot-backend/README.md

```diff
@@ -15,10 +15,15 @@ To configure the plugin using the new backend system:
 
    const backend = createBackend();
 
-   backend.add(import('@backstage-community/plugin-copilot'));
+   backend.add(import('@backstage-community/plugin-copilot-backend'));
```

Comment:
Since we are here we could make it a diff :D 

```diff
  import { createBackend } from '@backstage/backend-defaults';

  const backend = createBackend();

+ backend.add(import('@backstage-community/plugin-copilot-backend'));

  backend.start();
```